scripts/adlsgen2setup.ps1

```diff
@@ -0,0 +1,47 @@
+## Set the preference to stop on the first error
+$ErrorActionPreference = "Stop"
+
+Write-Host ""
+Write-Host "Loading azd .env file from current environment"
+Write-Host ""
+
+$output = azd env get-values
+
+foreach ($line in $output) {
```

Comment:
As youre now repeating this code in a few files, can you do what I did in the AAD PR and move into a loadenv.sh and loadenv.ps1 script?

Reply:
Great idea

Reply:
copied from your PR